papas/mergedclusterbuilder.py

```diff
@@ -67,14 +67,18 @@ def __init__(self, clusters, ruler, history_nodes = None):
     def _make_merged_clusters(self):
         #carry out the merging of linked clusters
         for subgraphids in self.subgraphs:
-            subgraphids.sort()
+            subgraphids.sort(reverse=True) #start with biggest clusters 
             first = None
             supercluster =None
             snode = None
+            totalenergy = 0.
+            #we want the merged cluster to have an identifier that matches its total energy
+            for elemid in subgraphids :
+                totalenergy += self.clusters[elemid].energy
             for elemid in subgraphids :
                 if not first:
                     first = elemid
-                    supercluster = MergedCluster(self.clusters[elemid])
+                    supercluster = MergedCluster(self.clusters[elemid], totalenergy)
```

Comment:
Perhaps I should pass the list of all clusters direct to the MergedCluster constructor rather than adding them in one by one afterwards. That could be nicer...